server/src/routes/getIngredientsById.ts

```diff
@@ -0,0 +1,33 @@
+import express, { RequestHandler } from "express";
+import { errors } from "@elastic/elasticsearch";
+import { elasticClient } from "../services/elasticClient";
+
+const router = express.Router();
+
+const routeHandler: RequestHandler = async (req, res) => {
+  try {
+    const query = {
+      index: "recipes",
+      id: req.params.id,
+    };
+
+    const result = await elasticClient.get(query);
+
+    res.json(result._source);
+  } catch (err) {
+    console.log("Error getting ingredients by mealId", err);
+    // if err is an object of the class ResponseError
```

Comment:
the parameter is called id not `mealId`
```suggestion
    console.log("Error getting ingredients by id", err);
    // if err is an object of the class ResponseError
```

Reply:
updated from `mealId` to `id` and committed changes